waltz-test/src/main/python/waltz_ducktape/tests/validation/scalability_test.py

```diff
@@ -28,6 +31,15 @@ def test_scale_up_replica(self, num_active_partitions, txn_per_client, num_clien
         self.run_produce_consume_validate(lambda: self.scale_up_replica(src_node_idx, added_node_idx, num_active_partitions,
                                                                         txn_per_client, num_clients, interval, timeout))
 
+    @cluster(cluster_spec=MIN_CLUSTER_SPEC)
+    @parametrize(num_active_partitions=4, txn_per_client=500, num_clients=1, interval=100, timeout=240, num_added_nodes=1, delay_before_torture=20)
+    @parametrize(num_active_partitions=1, txn_per_client=200, num_clients=1, interval=200, timeout=240, num_added_nodes=1, delay_before_torture=10)
+    def test_scale_up_server_nodes(self, num_active_partitions, txn_per_client, num_clients, interval, timeout,
+                                   num_added_nodes, delay_before_torture):
+        self.shrink_and_reallocate_waltz_server(num_added_nodes)
```

Comment:
Instead of this new method, why not just set `num_server_nodes` in ProduceConsumeValidateTest to the required number of server nodes that we want the Waltz cluster to start with?